### ISSUE TITLE
Too many neutral mobs

### ISSUE THREAD

**Issue by MsgLosers** [dalaranwow/dalaran-wow#5671]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
A lot of the mobs in Draenei starting are are Neutral. I'm sure both the mean flowers (Mutated Root Lashers) and the Blood Elf Scouts that just killed our dear Draenei scout (quest) are indeed not neutral but frikkin evil as heck. Can you please correct this? Thanks

P.S. Even the first quest about the Blood elves state that they are not neutral, but enemy:
I quote: *"You must eliminate...the blood elf... threat." *

**Expected behaviour**: Tell us what should happen instead
The weird plant things (Mutated Root Lashers) and the Blood Elf Scouts should be tagged as enemy.

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**
![alt text](https://i.paste.pics/799d2a7af4656934e2f4088be069b7b3.png "Proof")
https://i.paste.pics/799d2a7af4656934e2f4088be069b7b3.png
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Chenocide** [comment]
I didn't look up the mobs but it's normal for creatures to be neutral in starting zones.

**Comment by MsgLosers** [comment]
> I didn't look up the mobs but it's normal for creatures to be neutral in starting zones.

I just checked with other starting-area mobs that I know have been listed as "enemy" since the start of WoW and they are indeed also Neutral... Is this a a change that came with WotLK or something? I'm really surprised by it. I checked out the Blizz forums and people are indeed pointing out that starting with WotLK (but worse in Cata, but worst on MoP) players got buffer and NPCs/mobs squishier to ridiculous levels.

I'm stunned. I thought WoW was a good challenge at least up until MoP but I was obviously wrong. :(

Perhaps we need a x0.5 rate server for us Vanilla-plebs? ;D
